firmware/controllers/algo/engine_configuration.cpp

```diff
@@ -1080,8 +1080,8 @@ void setDefaultFrankensoConfiguration(DECLARE_CONFIG_PARAMETER_SIGNATURE) {
 //	engineConfiguration->gps_rx_pin = GPIOB_7;
 //	engineConfiguration->gps_tx_pin = GPIOB_6;
 
-	engineConfiguration->triggerSimulatorPins[0] = GPIOD_1;
-	engineConfiguration->triggerSimulatorPins[1] = GPIOD_2;
+	engineConfiguration->triggerSimulatorPins[0] = GPIOB_14;
```

Comment:
why and what's that? would I need to change my HW CI jumpers? W?

Reply:
@mck1117 ?

Reply:
you don't, that snuck in because I was testing on an MCU that doesn't actually have D1/2